embedchain/embedchain.py

```diff
@@ -380,7 +383,10 @@ def load_and_embed_v2(
             data_dict = {id: value for id, value in data_dict.items() if id not in existing_ids}
 
             if not data_dict:
-                print(f"All data from {src} already exists in the database.")
+                srcCopy = src
```

Comment:
This logic looks good.

Nit pick: can you follow snake case convention for variable naming?

Reply:
Done